### DISCUSSION TITLE
Folder groups and sidebar navigation

### DISCUSSION THREAD

**Discussion by Vanja-S** [fuma-nama/fumadocs#1173] [Q&A] [answered]
Hi!

Following the logic and docs for the folder groups feature: https://fumadocs.vercel.app/docs/ui/page-conventions#folder-group 
I constructed the following content folder:

```ascii
content/
└── (static-pages)/
    ├── cookies.md
    ├── legal.md
    ├── gdpr.md
    └── ...
```

this works (at least it allows for the slug to be unchanged) but does not render the sidebar file tree (the fumadocs provided sidenav) correctly, when visiting any of these pages (cookies, legal..._) - assuming that is because its parent (the folder) is not visible, but changing to:

```ascii
content/
└── (static-pages)/
    └── legal/
        ├── cookies.md
        ├── legal.md
        ├── gdpr.md
        └── ...
``` 
still does not render the correct file tree, rather it only renders the other top-level folders and docs from the content folder. Have I misunderstood the folder group feature? Is there a way to force render the contents of (static-page)?

Thanks!

**Comment by fuma-nama** [comment]
Hmm `content/docs` is the only scanned directory by default, you should put all content files inside

**Reply by Vanja-S** [reply]
True, but (I forgot to mention in the org post) you can configure that inside source.config.ts:
```ts
export const { docs, meta } = defineDocs({
  dir: 'content',
  docs: {
    schema: frontmatterSchema.extend({
      'adi.editUrl': z.string().optional()  
    })
  }
});
```

where only content is scanned.
Other pages are correctly displayed - and so are their ToCs in the sidebar - I only have a problem with the specified setup, where I would want to group a few md files.

**Reply by fuma-nama** [reply, marked as answer]
You probably forgot to add the folder to meta.json, it is hidden unless you specify it or use the rest operator.

**Reply by Vanja-S** [reply]
That did the trick. Thanks for the quick answer!
